test/monster.jl

```diff
@@ -1,4 +1,8 @@
 module Example
+# Compat
+if !isdefined(Core, :String)
+    typealias String UTF8String
+end
```

Comment:
I'd rather put this definition in the `header.jl` file for now. That way it's a little more hidden from the user.